.gitignore

```diff
@@ -208,6 +208,7 @@ cmake-build-debug/
 /test/memory/shannon/jacobi.log
 /test/memory/shannon/configVarFile.log
 
+/test/library/packages/Socket.compopts
```

Comment:
For future reference, we're trying to get into the habit of using directory-specific .gitignore files so that if a directory is renamed, moved, or deleted, the .gitignore will go along with it rather than requiring maintenance in the top-level file or resulting in zombie entries.  I've created https://github.com/chapel-lang/chapel/issues/20263 to help avoid this trap (you're not the first to have stepped into it).